cypress/e2e/createEditTenure.feature

```diff
@@ -1,8 +1,11 @@
+@tenure
 @authentication
 @common
-@processes
 @root
-@tenure
+@search
+@worktray
+@processes
+@personal-details
```

Comment:
Again, it's not intuitive, but the `add person to tenure` is part of `tenure` MFE functionality, not `person-details`. Same for creating the [person](https://github.com/LBHackney-IT/mtfh-frontend-tenure/blob/3c8f485d9075d3ed645dc4f50b6181cf52e3b35d/src/views/add-person-to-tenure-view/add-or-remove-person-from-tenure-view.tsx) - it's part of `tenure` MFE, these tests should be uncommented:
![image](https://github.com/LBHackney-IT/mtfh-tl-e2e-tests/assets/43747286/b5d65a0d-f4eb-4916-add2-743571df48bb)

So I don't think this spec should be run when `personal-details` MFE receives changes.